docs/.vuepress/.temp/pages/docs/atom-archive/getting-started/sections/why-atom.html.vue

```diff
@@ -0,0 +1,25 @@
+<template><div><h3 id="why-atom" tabindex="-1"><a class="header-anchor" href="#why-atom" aria-hidden="true">#</a> Why Atom?</h3>
+<p>There are a lot of text editors out there; why should you spend your time learning about and using Atom?</p>
+<p>Editors like Sublime and TextMate offer convenience but only limited extensibility. On the other end of the spectrum, Emacs and Vim offer extreme flexibility, but they aren't very approachable and can only be customized with special-purpose scripting languages.</p>
+<p>We think we can do better. Our goal is a zero-compromise combination of hackability and usability: an editor that will be welcoming to an elementary school student on their first day learning to code, but also a tool they won't outgrow as they develop into seasoned hackers.</p>
+<p>As we've used Atom to build Atom, what began as an experiment has gradually matured into a tool we can't live without. On the surface, Atom is the modern desktop text editor you've come to expect. Pop the hood, however, and you'll discover a system begging to be hacked on.</p>
+<h4 id="the-nucleus-of-atom" tabindex="-1"><a class="header-anchor" href="#the-nucleus-of-atom" aria-hidden="true">#</a> The Nucleus of Atom</h4>
```

Comment:
Again, concerns about heading levels, though I'm not sure how this file is used.